code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -60,11 +60,11 @@ Difficulty: Medium
 	medal_type = BOSS_MEDAL_DRAKE
 	score_type = DRAKE_SCORE
 	deathmessage = "collapses into a pile of bones, its flesh sloughing away."
-	death_sound = 'sound/magic/demon_dies.ogg'
+	deathsound = 'sound/magic/demon_dies.ogg'
 	var/datum/action/small_sprite/smallsprite = new/datum/action/small_sprite/drake()
-	
+
```

Comment:
accidentally deleted a bunch of whitespace in this file, can undo if maintainers want